packages/test-commons/src/utils.ts

```diff
@@ -116,25 +114,20 @@ export function createMouseEvent(eventType = "click", clientX = 0, clientY = 0)
             break;
     }
 
+    const event = new MouseEvent(eventType, {
+        bubbles: true,
+        button: 0,
+        cancelable: true,
+        clientX,
+        clientY,
+        detail: detailArg,
+        view: window,
+    });
+    // https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail
+
     // HACKHACK: see https://github.com/palantir/blueprint/issues/5173
     // eslint-disable-next-line deprecation/deprecation
```

Comment:
wrong location for this MDN link, also we should remove the HACKHACK and eslint comment flag

```suggestion

```